sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkOptions.java

```diff
@@ -426,4 +427,12 @@
   String getBootstrapServers();
 
   void setBootstrapServers(String value);
+
+  @Description("Same as --numWorkers in DataflowPipelineWorkerPoolOptions")
```

Comment:
What does these options mean for non Dataflow runners? Is there a more generic way to define them?

Reply:
They will be ignored I think. I don't know if there is any better way to define them. Note that we need to look at it only to override them for generator pipeline.. if we are ok using same options for generator and the main pipeline, we don't need these options.